.github/contribuiting.md

```diff
@@ -0,0 +1,36 @@
+# 👍 Contribute
+
+If you want to say **thank you** and/or support the active development of `Fiber`:
+
+1. Add a [GitHub Star](https://github.com/gofiber/fiber/stargazers) to the project.
+2. Tweet about the project [on your Twitter](https://twitter.com/intent/tweet?text=%F0%9F%9A%80%20Fiber%20%E2%80%94%20is%20an%20Express.js%20inspired%20web%20framework%20build%20on%20Fasthttp%20for%20%23Go%20https%3A%2F%2Fgithub.com%2Fgofiber%2Ffiber).
+3. Write a review or tutorial on [Medium](https://medium.com/), [Dev.to](https://dev.to/) or personal blog.
+4. Help us to translate this `README` to another language.
+5. Support the project by donating a [cup of coffee](https://buymeacoff.ee/fenny).
+
+## Commits
+Based on https://chris.beams.io/posts/git-commit/
+
+1. Separate subject from body with a blank line
+2. Limit the subject line to 50 characters
+3. Capitalize the subject line
+4. Do not end the subject line with a period
+5. Use the imperative mood in the subject line
+6. Wrap the body at 72 characters
+7. Use the body to explain what and why vs. how
+
+## Prefix to use in titles of your pull requests or commits
+We will see now some examples of titles always we must use prefix according to below:
+> 🔥Feature, ♻️Refactor, 🩹Fix, 🚨Test, 📚Doc, 🎨Style
+- 🔥Feat: Add flow to add person
+- ♻️Refactor: Rename file X to Y
+- 🩹Fix: Improve flow
+- 🚨Test: Validate to add a new person
+- 📚Doc: Translate to Portuguese middleware redirect
+- 🎨Style: Respected pattern Golint
+
+All pull request that are a feature or fix is mandatory to have unit tests on the pull request.
+
+
+## Note
+Your pull request only to merge if you respecting this flow, even so, is required minimal 2 approval from members to Fiber.
```

Comment:
## Prefixes to use in titles of your pull requests or commits
We will see now some examples of titles using the following mandatory prefixes:
> 🔥Feature, ♻️Refactor, 🩹Fix, 🚨Test, 📚Doc, 🎨Style

- 🔥Feat: Improve URL matcher
- ♻️Refactor: Rename file X to Y
- 🩹Fix: Improve flow
- 🚨Test: Write unit tests for the URL matcher
- 📚Doc: Translate the redirect middleware to Portuguese
- 🎨Style: Follow Golint pattern

All pull request that implement a feature or fix are required to have unit tests in the pull request.


## Note
Your pull request will only be merged if you respect this flow. In addition, it must get at least 2 approvals from members of the Fiber team.